src/components/ui/popover.tsx

```diff
@@ -49,6 +49,13 @@ export const PopoverCloseTrigger = React.forwardRef<
   );
 });
 
+export const PopoverTriggerWithClear = React.forwardRef<
+  HTMLButtonElement,
+  ChakraPopover.TriggerProps
+>(function PopoverTriggerWithClear(props, ref) {
+  return <ChakraPopover.Trigger {...props} ref={ref} />;
+});
+
```

Comment:
would it work with something like:
```suggestion
export const PopoverTrigger = ChakraPopover.Trigger;
```

Reply:
This isn't used anywhere. I think I was just playing around with an idea